src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/documents/recallnotification/RevocationOrderServiceTest.kt

```diff
@@ -87,6 +90,62 @@ internal class RevocationOrderServiceTest {
       }.verifyComplete()
   }
 
+  @Test
+  fun `creates a new revocation order for a recall`() {
```

Comment:
Personally I find these test names are only clear when they differentiate the function under test, so this might be:
`generateAndStorePdf creates a new revocation order for a recall`,
whilst e.g. the preceding one (in this file) becomes:
`getOrGeneratePdf creates a revocation order for a recall if one does not exist`